### PR TITLE
Update/clean lunch.rb

### PR DESCRIPTION
**Pull request by Levis92**

### CODE REVIEW

File: app/models/lunch.rb [hunk 1]
```diff
@@ -6,11 +6,13 @@ class << self
     include JSON
 
     # Allergies supported in API
+    # https://chalmerskonferens.se/en/api/
```
Comment (lindskogen) [inline app/models/lunch.rb]: Are we even using this API now?

File: app/models/lunch.rb [hunk 2]
```diff
@@ -62,17 +64,14 @@ def einstein
     end
 
     def chalmrest
-      date = Time.new.strftime("%Y-%m-%d")
-      locale = I18n.locale.to_s
```
Comment (lindskogen) [inline app/models/lunch.rb]: No locale support? :(

File: app/models/lunch.rb (hunk 1 above)
Comment (Levis92) [inline app/models/lunch.rb]: Yes

File: app/models/lunch.rb (hunk 2 above)
Comment (Levis92) [inline app/models/lunch.rb]: Is fixed 😃

Comment (Levis92) [inline app/models/lunch.rb]: Is `date = Time.new.strftime("%Y-%m-%d")` used for anything i Rails by the way? That variable didn't seem to be used.

File: app/models/lunch.rb [hunk 3]
```diff
@@ -85,18 +86,24 @@ def chalmrest
...
 
             category[recipes].map do |recipe|
               allergens = recipe[allergensKey]
-              summary = recipe[recipeNames].first[recipeName]
+              if locale == "sv"
+                displayName = category[name]
+                summary = recipe[recipeNames].first[recipeName]
+              else
+                displayName = category[nameEnglish]
+                summary = recipe[recipeNames][1][recipeName]
```
Comment (lindskogen) [inline app/models/lunch.rb]: `.second`? 😉

File: app/models/lunch.rb (hunk 1 above)
Comment (lindskogen) [reply app/models/lunch.rb]: I don't see where? 🤔

File: app/models/lunch.rb (hunk 3 above)
Comment (Levis92) [reply app/models/lunch.rb]: Didn't know that was a thing ^^ Will fix! 😉

Comment (Levis92) [reply app/models/lunch.rb]: It's where you find the links to the APIs for each restaurant.

File: app/models/lunch.rb (hunk 1 above)
Comment (Levis92) [reply app/models/lunch.rb]: It's where you find the links to the APIs for each restaurant.

**Pull request merged**